src/v/kafka/protocol/schemata/generator.py

```diff
@@ -473,6 +473,32 @@
     'creatable_topic_result': 'chunked_vector',
     'describe_configs_resource_result': 'chunked_vector',
     'describe_configs_result': 'chunked_vector',
+    'incremental_alterable_config': 'chunked_vector',
```

Comment:
we should probably remove the 'chunked_vector' overrides in the generator.py file

Reply:
Done